### PR TITLE
syncs up CAS with warehouse deployment

### PR DESCRIPTION
**Pull request by yakloinsteak**

### CODE REVIEW

**Comment by yakloinsteak** [comment]
@eanders I don't have a login to CAS, but I did deploy this to staging successfully.

File: config/brakeman.ignore [hunk 1]
```diff
@@ -152,6 +192,86 @@
...
+      "user_input": "`git rev-parse --abbrev-ref HEAD`.chomp",
+      "confidence": "Medium",
+      "note": "deployment code takes no outside inputs"
+    },
+    {
+      "warning_type": "HTTP Verb Confusion",
+      "warning_code": 118,
+      "fingerprint": "7452bf4ee658fb4a4704831cce06d767e0d9310795eb7f6c3b71cd3be36cca5c",
+      "check_name": "VerbConfusion",
+      "message": "Potential HTTP verb confusion. `HEAD` is routed like `GET` but `request.get?` will return `false`",
```
Comment (yakloinsteak) [inline config/brakeman.ignore]: Had to upgrade brakeman and then this warning came up. Feels like ignoring was the right fix.

Comment (eanders) [reply config/brakeman.ignore]: yep

**Comment by yakloinsteak** [comment]
@eanders Can we merge this this week some time do you think?

**Pull request merged**